source/reference/failure-recovery.txt

```diff
@@ -12,22 +12,28 @@ Recovery From Sync Failure
    :depth: 2
    :class: singlecol
 
+Setting up {+c2c-product-name+} to prepare for disaster recovery follows the 
+same procedure as other :ref:`connections <c2c-connecting>` between clusters.
+Synchronization keeps the destination cluster up to date with the source
+cluster and ready as a backup. In the event that the source cluster fails or
+becomes otherwise unavailable, the destination cluster can replace it as the
+primary cluster.
+
 :program:`mongosync` temporarily alters some collection characteristics
 during synchronization. The original values are restored during the
-:ref:`commit process <c2c-api-commit>`. However, a permanent system
-failure could disrupt synchronization before the sync is committed. If
-the sync ends prematurely, you must manually reset the collection
-characteristics on the destination cluster.
+:ref:`commit process <c2c-api-commit>`. If synchronization ends prematurely,
+you must manually reset these collection characteristics on the destination
+cluster before you can use it as the primary cluster.
+
+To prepare the destination cluster to replace the primary, make the following
+changes:
 
-Manually revert the following temporary changes:
+#. :ref:`Set Unique indexes as non-unique <c2c-dr-unique>`
+#. :ref:`Set TTL indexes with expireAfterSeconds to MAX_INT
+   <c2c-dr-ttl>`
+#. :ref:`Set Hidden indexes as non-hidden <c2c-dr-hidden>`
```

Comment:
These should be the other way around.
During C2C, mongosync does the following as part of its replication machinery

- Set Unique indexes as non-unique
- Set TTL indexes with expireAfterSeconds to MAX_INT
- Set Hidden indexes as non-hidden

But users need to effectively undo this to get back their unique indexes/TTL indexes/hidden indexes.

Reply:
Fixed